openlibrary/components/BulkSearch/utils/samples.js

```diff
@@ -0,0 +1,9 @@
+export const sampleData = {
```

Comment:
Great! This would be cleaner as:

```js
[
    {
        name: "Public domain 2023",
        source: "https://...",
        text: "Djamaluddin Adin..."
    },
    ...
]
```

And so on. That'll let us keep the links which is important, even if it's only in the code! And it'll make rendering them cleaner, since the name of the sample and the sample itself will be right next to each other, whereas currently the names are flung out in an entirely different file!

Reply:
Excellent suggestion, and fixed; I also added the default 'no sample' option to the array.